src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
```

Comment:
The border color should be consistent with the design. Please ensure that the border color matches the color specified in the design requirements.